src/app/[locale]/(auth)/dashboard/user-profile/[[...user-profile]]/page.tsx

```diff
@@ -14,7 +14,7 @@ export async function generateMetadata(props: { params: { locale: string } }) {
 
 const UserProfilePage = () => (
   <div className="my-6 -ml-16">
-    <UserProfile />
+    <UserProfile path="/dashboard/user-profile" />
```

Comment:
How about adding path in SignIn and SignUp component? https://github.com/ixartz/Next-js-Boilerplate/blob/main/src/app/%5Blocale%5D/(auth)/(center)/sign-in/%5B%5B...sign-in%5D%5D/page.tsx#L16 and https://github.com/ixartz/Next-js-Boilerplate/blob/main/src/app/%5Blocale%5D/(auth)/(center)/sign-up/%5B%5B...sign-up%5D%5D/page.tsx

Based on the CHANGELOG example: https://github.com/clerk/javascript/blob/main/packages/nextjs/CHANGELOG.md#major-changes

Reply:
Yeah let me double check they are required on those as well. It was surfaced that their should be some sort of TS error or something being required for `UserProfile`

Reply:
Yep its needed on all them, nice catch 😆

Reply:
Awesome, you commit the changes.

Reply:
I did 😄